### PR TITLE
modulesync 2.6.0

### PR DESCRIPTION
**Pull request by Dan33l**

### CODE REVIEW

File: spec/acceptance/nodesets/archlinux-2-x64.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
[the remark this answers is not in the document]
Comment (Dan33l) [reply spec/acceptance/nodesets/archlinux-2-x64.yml]: I was think about you when i saw the nodesets :+1: 
I pushed to update.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,26 +1,38 @@
-require 'rubygems'
```
Comment (alexjfisher) [inline spec/spec_helper.rb]: @Dan33l For the initial modulesync, perhaps mark this file as unmanaged?  A future PR can update it to the one from modulesync_config.

File: .travis.yml [hunk 3]
```diff
@@ -1,18 +1,47 @@
...
+  irc:
+    on_success: always
+    on_failure: always
+    channels:
+      - "chat.freenode.org#voxpupuli-notifications"
+deploy:
+  provider: puppetforge
+  user: puppet
+  password:
+    secure: ""
```
Comment (alexjfisher) [inline .travis.yml]: You could also add the secret from plumbing to `.sync.yml` and regenerate this file.

**Comment by bastelfreak** [comment]
I added the missing travis secret. I'm going to merge this because it's blocking PRs on modulesync_config.

**Comment by bastelfreak** [comment]
Follow up: https://github.com/voxpupuli/puppet-keepalived/issues/156

**Pull request merged**